docs/upgrade.md

```diff
@@ -55,6 +55,10 @@ Bubbled child events no longer pass the `childView` implicitly and only pass the
 arguments passed as part of `triggerMethod`. This means that the arguments
 passed to `onEvent` and `onChildviewEvent` are now identical.
 
+In Marionette 2, `childEvents` where bound on every event. In Marionette 3,
+`childViewEvents` are bound once and cached. This means that you cannot add new
```

Comment:
That's sounds a lot more explicit. I just noticed that > where bound

Should be

> were bound

I suggest mentioning that it is still possible to bind events outside of using the hash. I can provide an example tomorrow. Typing this on my phone.